website/docs/getting-started/setup/installation-guides/kubernetes/README.mdx

```diff
@@ -473,30 +499,33 @@ When you run the install command, you may see output as shown below:
   STATUS: deployed
   REVISION: 1
   TEST SUITE: None
-  NOTES:
-  1. Get the application URL by running these commands:
-    export POD_NAME=$(kubectl get pods --namespace default -l "app.kubernetes.io/name=appsmith,app.kubernetes.io/instance=appsmith-2-1670479627" -o jsonpath="{.items[0].metadata.name}")
-    export CONTAINER_PORT=$(kubectl get pod --namespace default $POD_NAME -o jsonpath="{.spec.containers[0].ports[0].containerPort}")
-    echo "Visit http://127.0.0.1:8080 to use your application"
-    kubectl --namespace default port-forward $POD_NAME 8080:$CONTAINER_PORT
-
-  To expose your Appsmith service to be accessible from the Internet, please refer to our docs here https://docs.appsmith.com/setup/kubernetes#expose-appsmith.
   ```
 
-2. Run the command `kubectl get pods` to verify the list of pods. You may see output as shown below:
+  Once deployed run the below commands to verify the installation.
 
-```
-NAME                            READY   STATUS    RESTARTS   AGE
-appsmith-875b6cddc-4mmj6        1/1     Running   0          90s
-appsmith-875b6cddc-7b2hw        1/1     Running   0          3m30s
-appsmith-875b6cddc-smzwz        1/1     Running   0          3m15s
-appsmith-mongodb-0              1/1     Running   0          3m30s
-appsmith-mongodb-1              1/1     Running   0          3m10s
-appsmith-mongodb-arbiter-0      1/1     Running   0          3m30s
-appsmith-postgresql-0           1/1     Running   0          3m30s
-appsmith-redis-master-0         1/1     Running   0          3m30s
-appsmith-redis-replicas-0       1/1     Running   0          3m30s
-```
+ i. To get the application URL run the below commands:
+  
+  ```
+  export POD_NAME=$(kubectl get pods --namespace default -l "app.kubernetes.io/name=appsmith,app.kubernetes.io/instance=appsmith-2-1670479627" -o jsonpath="{.items[0].metadata.name}")
+  export CONTAINER_PORT=$(kubectl get pod --namespace default $POD_NAME -o jsonpath="{.spec.containers[0].ports[0].containerPort}")
+  echo "Visit http://127.0.0.1:8080 to use your application"
+  kubectl --namespace default port-forward $POD_NAME 8080:$CONTAINER_PORT
+  ```
+
+ii. Run the command `kubectl get pods` to verify the list of pods. You may see output as shown below:
+
+  ```bash
+  NAME                            READY   STATUS    RESTARTS   AGE
+  appsmith-875b6cddc-4mmj6        1/1     Running   0          90s
+  appsmith-875b6cddc-7b2hw        1/1     Running   0          3m30s
+  appsmith-875b6cddc-smzwz        1/1     Running   0          3m15s
+  appsmith-mongodb-0              1/1     Running   0          3m30s
+  appsmith-mongodb-1              1/1     Running   0          3m10s
+  appsmith-mongodb-arbiter-0      1/1     Running   0          3m30s
+  appsmith-postgresql-0           1/1     Running   0          3m30s
+  appsmith-redis-master-0         1/1     Running   0          3m30s
+  appsmith-redis-replicas-0       1/1     Running   0          3m30s
+  ```
```

Comment:
Here also, both listing syntax, as well as alignment, are off. Just check out how this renders. The nested list hierarchy is completely lost.

![Screenshot 2023-01-03 at 4 48 35 PM](https://user-images.githubusercontent.com/120119/210347562-3462e7ac-3176-495c-bdd2-9cdbdb9df4f6.png)